health-management-information-system/kubernetes/main/k8s.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+k8sMainRootFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+
```

Comment:
@bradsawadye your k8s service is working well 👍🏼 One improvement though would be to print out the service URLs for minikube in the script - it really makes testing easier. The core script outputs the URLs really well: https://github.com/openhie/instant/blob/master/core/kubernetes/main/k8s.sh

Reply:
addresed